README.md

```diff
@@ -38,3 +38,14 @@ Metacello new
   repository: 'github://mumez/RediStick/repository';
   load: #('Core' 'ConnectionPool').
 ```
+
+## how to setUp test environement
+
+in this repository there is a redis.conf file , redis use it for initialisation
+
+```bash
```

Comment:
I decide to not setup automatically a test environment , I suppose that the user will set up by himself the Redis database

Reply:
I think we can propose those 2 solutions (with/without docker ) , I will update readme in a next commit

Reply:
fix with commit  36133d3 and  2bdbdd4